lib/styles/app_theme.dart

```diff
@@ -1,3 +1,16 @@
+import 'package:flutter/material.dart';
+
 class AppTheme {
   AppTheme._();
-}
\ No newline at end of file
+
+  //const values
+  // static const Color _lightprimarycolur = Colors.white;
+
+  static final ThemeData lighttheme = ThemeData(
+    primarySwatch: Colors.blue,
+  );
+
+  static final ThemeData darktheme = ThemeData(
+    brightness: Brightness.dark,
+  );
```

Comment:
@AmanKumarSingh01 How is this is called theming?

Reply:
I saw this on several websites and youtube videos,

Could you be more specific what you want?

Through the file we can manage themes across the app.

https://www.youtube.com/watch?v=RJkiESVJXAk&t=1439s check this youtube video

Reply:
https://techieblossom.com/theming-in-flutter/